code/kg2/filter_kg_and_remap_predicates.py

```diff
@@ -151,4 +152,11 @@ def make_arg_parser():
             print('relation curie is in the config file but was not detected in the graph: ' + relation_curie, file=sys.stderr)
     for relation_curie in relation_curies_not_in_nodes:
         print('could not get IRI for relation curie: ' + relation_curie)
+    build_info = {
+        "name" : "KG2:Build",
+        "creation date" : datetime.now().strftime("%m-%d-%Y"),
+        "category label" : "information content entity",
```

Comment:
@saramsey I forgot to resolve that comment, so it was showing you an old version of the file changes. this is what will be merged :)